core/src/main/java/org/mskcc/cbio/portal/scripts/ImportClinicalData.java

```diff
@@ -104,7 +124,7 @@ public void importData() throws Exception
         FileReader reader =  new FileReader(clinicalDataFile);
         BufferedReader buff = new BufferedReader(reader);
         List<ClinicalAttribute> columnAttrs = grabAttrs(buff);
-
+        
         int patientIdIndex = findPatientIdColumn(columnAttrs);
         int sampleIdIndex = findSampleIdColumn(columnAttrs);
 
```

Comment:
this check on row 131 needs to be adjusted: PATIENT_ID always required, SAMPLE_ID is only required when `this.attributesDatatype.equals("SAMPLE_ATTRIBUTES")`